web/models/meeting_type.ex

```diff
@@ -7,14 +7,16 @@ defmodule Meetings.MeetingType do
     field :title, :string
     field :description, :string
     field :location, :string
+    field :building, :string
```

Comment:
What's the building field for? And is it common enough in meetings that it constitutes having a field in the main schema? Or could this be something covered with `meeting_extra`?